### PR TITLE
fix (DB/Loot) Add Tailoring condition for the Battlecast Hood Recipe

### PR DESCRIPTION
**Pull request by Tereneckla**
## Changes Proposed:
-  Add missing condition to require tailoring for the [battlecast hood recipe](https://wowgaming.altervista.org/aowow/?item=24313) from Warlord Kalithresh
-  

## Issues Addressed:

- Closes 

## SOURCE:

Common knowledge that the bop recipes in TBC require the profession
Comments: https://www.wowhead.com/wotlk/item=24313/pattern-battlecast-hood#comments

## Tests Performed:

- Did test as described below
- 

## How to Test the Changes:

```UPDATE `creature_loot_template` SET `Chance`=100 WHERE `Item`=24313;```
Killed Warlord Kalithresh with and without tailoring

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1680525460717061700.sql [hunk 1]
```diff
@@ -0,0 +1,2 @@
+INSERT INTO `conditions` (`SourceTypeOrReferenceId`, `SourceGroup`, `SourceEntry`,`ConditionTypeOrReference`, `ConditionValue1`, `ConditionValue2`) VALUES
+(1, 17798, 24313, 7, 197, 1);
```
Comment (Gultask) [inline data/sql/updates/pending_db_world/rev_1680525460717061700.sql:2]:
```suggestion
(1, 17798, 24313, 7, 197, 1);

```

Comment (Gultask) [inline data/sql/updates/pending_db_world/rev_1680525460717061700.sql:2]: Missing DELETE query and newline at the end

**Review by Gultask — CHANGES_REQUESTED** [review]

**Comment by neifion-00000000** [comment]
thanks for getting this, note for whoever deals with this next, the drop rate is definitely too low, but there's no way to properly source the drop rate other than grinding this boss as a tailor

File: data/sql/updates/pending_db_world/rev_1680525460717061700.sql (hunk 1 above)
Comment (Tereneckla) [reply data/sql/updates/pending_db_world/rev_1680525460717061700.sql]: There was no previous condition for Kalithresh Normal, but sure

**Review by Gultask — APPROVED** [review]

File: data/sql/updates/pending_db_world/rev_1680525460717061700.sql [hunk 2]
```diff
@@ -0,0 +1,3 @@
+DELETE FROM `conditions` WHERE `SourceTypeOrReferenceId` = 1 AND `SourceGroup` = 17798 AND `SourceEntry` = 24313 AND `ConditionTypeOrReference` = 7;
+INSERT INTO `conditions` (`SourceTypeOrReferenceId`, `SourceGroup`, `SourceEntry`,`ConditionTypeOrReference`, `ConditionValue1`, `ConditionValue2`) VALUES
+(1, 17798, 24313, 7, 197, 1);
```
Comment (Gultask) [inline data/sql/updates/pending_db_world/rev_1680525460717061700.sql:3]: Oh, oops, I didn't notice earlier but it's missing comment

**Review by Gultask — CHANGES_REQUESTED** [review]

File: data/sql/updates/pending_db_world/rev_1680525460717061700.sql (hunk 2 above)
Comment (Tereneckla) [reply data/sql/updates/pending_db_world/rev_1680525460717061700.sql:3]: I skipped the comment field because there is no row in conditions with a comment

**Review by Gultask — APPROVED** [review]

**Comment by Gultask** [comment]
Awesome, ty!

**Pull request merged**